src/openklant/utils/tests/test_validators.py

```diff
@@ -147,3 +148,22 @@ def test_validate_iban(self):
         self.assertIsNone(validate_iban("ab1299999999999"))
         self.assertIsNone(validate_iban("ab129"))
         self.assertIsNone(validate_iban("ab12aaaaaaaaaa"))
+
+    def test_validate_country(self):
+        invalid_codes = [
+            "",
+            "1",
+            "10",
+            "ZZ",
+            "1Z",
+        ]
+        for code in invalid_codes:
+            self.assertRaisesMessage(
+                ValidationError,
+                "Ongeldige landcode, de code moet behoren tot de ISO 3166-standaard",
+                validate_country,
+                code,
+            )
+
+        self.assertIsNone(validate_country("NL"))
+        self.assertIsNone(validate_country("nl"))
```

Comment:
This should raise errors (like I mentioned in `validators.py`), so this can be added to `invalid_codes` once that feedback has been addressed